pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 05:23:59.127692 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.session.session - PingTask stopped
pyrogram.connection.connection - Disconnected
pyrogram.session.session - NetworkTask stopped
pyrogram.session.session - Session stopped
pyrogram.session.auth - Start creating a new auth key on DC2
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC2 - IPv4 - TCPAbridgedO
pyrogram.session.auth - Done auth key exchange: DhGenOk
pyrogram.connection.connection - Disconnected
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC2 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 05:25:11.774201 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.session.session - PingTask stopped
pyrogram.connection.connection - Disconnected
pyrogram.session.session - NetworkTask stopped
pyrogram.session.session - Session stopped
pyrogram.session.auth - Start creating a new auth key on DC2
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC2 - IPv4 - TCPAbridgedO
pyrogram.session.auth - Done auth key exchange: DhGenOk
pyrogram.connection.connection - Disconnected
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC2 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 05:26:41.306120 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.session.session - PingTask stopped
pyrogram.connection.connection - Disconnected
pyrogram.session.session - NetworkTask stopped
pyrogram.session.session - Session stopped
pyrogram.session.auth - Start creating a new auth key on DC2
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC2 - IPv4 - TCPAbridgedO
pyrogram.session.auth - Done auth key exchange: DhGenOk
pyrogram.connection.connection - Disconnected
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC2 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 05:30:37.527402 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.session.session - PingTask stopped
pyrogram.connection.connection - Disconnected
pyrogram.session.session - NetworkTask stopped
pyrogram.session.session - Session stopped
pyrogram.session.auth - Start creating a new auth key on DC5
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.auth - Done auth key exchange: DhGenOk
pyrogram.connection.connection - Disconnected
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
root - Stop signal received (SIGINT). Exiting...
__main__ - <---Bot Stopped-->
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 05:34:24.429889 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 05:34:59.984226 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
root - Stop signal received (SIGINT). Exiting...
__main__ - <---Bot Stopped-->
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 05:44:11.016607 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
pyrogram.dispatcher - type object 'Message' has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, Message.id),
                                            ^^^^^^^^^^
AttributeError: type object 'Message' has no attribute 'id'
root - Stop signal received (SIGINT). Exiting...
__main__ - <---Bot Stopped-->
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 05:45:19.740146 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
root - Stop signal received (SIGINT). Exiting...
__main__ - <---Bot Stopped-->
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.connection.connection - Disconnected
pyrogram.session.session - NetworkTask stopped
pyrogram.session.session - Session stopped
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 05:46:15.561032 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
pyrogram.dispatcher - tuple index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 195, in check
    listener = client.match_listener(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 133, in match_listener
    elif (data[0], None, data[2]) in listeners:
                         ~~~~^^^
IndexError: tuple index out of range
root - Stop signal received (SIGINT). Exiting...
root - Stop signal received (SIGINT). Exiting...
root - Stop signal received (SIGINT). Exiting...
root - Stop signal received (SIGINT). Exiting...
__main__ - <---Bot Stopped-->
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 05:47:39.180470 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
pyrogram.dispatcher - 'Message' object has no attribute 'id'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 208, in handler_worker
    if await handler.check(self.client, parsed_update):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 196, in check
    (message.chat.id, message.from_user.id, message.id),
                                            ^^^^^^^^^^
AttributeError: 'Message' object has no attribute 'id'
root - Stop signal received (SIGINT). Exiting...
__main__ - <---Bot Stopped-->
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 05:48:53.279418 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - cannot access local variable 'token' where it is not associated with a value
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\khan.py", line 67, in account_login
    'authorization': f"Bearer {token}",
                     ^^^^^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'token' where it is not associated with a value
pyrogram.dispatcher - cannot access local variable 'token' where it is not associated with a value
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\khan.py", line 67, in account_login
    'authorization': f"Bearer {token}",
                     ^^^^^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'token' where it is not associated with a value
pyrogram.dispatcher - 'data'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\pw.py", line 107, in account_login
    response2 = requests.get(f'https://api.penpencil.xyz/v3/batches/{raw_text3}/details', headers=headers).json()["data"]["subjects"]
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^
KeyError: 'data'
pyrogram.dispatcher - cannot access local variable 'token' where it is not associated with a value
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\khan.py", line 67, in account_login
    'authorization': f"Bearer {token}",
                     ^^^^^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'token' where it is not associated with a value
pyrogram.dispatcher - list index out of range
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\khan.py", line 55, in account_login
    info["password"] = raw_text.split("*")[1]
                       ~~~~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
pyrogram.dispatcher - cannot access local variable 'token' where it is not associated with a value
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\khan.py", line 67, in account_login
    'authorization': f"Bearer {token}",
                     ^^^^^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'token' where it is not associated with a value
pyrogram.dispatcher - 'data'
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\pw.py", line 107, in account_login
    response2 = requests.get(f'https://api.penpencil.xyz/v3/batches/{raw_text3}/details', headers=headers).json()["data"]["subjects"]
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^
KeyError: 'data'
pyrogram.dispatcher - cannot access local variable 'token' where it is not associated with a value
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\khan.py", line 67, in account_login
    'authorization': f"Bearer {token}",
                     ^^^^^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'token' where it is not associated with a value
root - Stop signal received (SIGINT). Exiting...
__main__ - <---Bot Stopped-->
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 06:07:34.933706 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - cannot access local variable 'token' where it is not associated with a value
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\cw.py", line 106, in account_login
    await m.reply_text(f"```{token}```")
                       ^^^^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'token' where it is not associated with a value
root - Stop signal received (SIGINT). Exiting...
__main__ - <---Bot Stopped-->
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 06:09:33.651816 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\requests\models.py", line 971, in json
    return complexjson.loads(self.text, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\cw.py", line 109, in account_login
    html1 = s.get("https://elearn.crwilladmin.com/api/v1/comp/my-batch?&token=" + token).json()
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\requests\models.py", line 975, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
root - Stop signal received (SIGINT). Exiting...
__main__ - <---Bot Stopped-->
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 07:17:06.619834 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - cannot access local variable 'token' where it is not associated with a value
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\cw.py", line 106, in account_login
    await m.reply_text(f"```{token}```")
                       ^^^^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'token' where it is not associated with a value
root - Stop signal received (SIGINT). Exiting...
__main__ - <---Bot Stopped-->
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 07:21:06.007034 UTC
pyrogram.connection.connection - Disconnected
pyrogram.session.session - NetworkTask stopped
pyrogram.session.session - Session stopped
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\requests\models.py", line 971, in json
    return complexjson.loads(self.text, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\cw.py", line 109, in account_login
    html1 = s.get("https://elearn.crwilladmin.com/api/v1/comp/my-batch?&token=" + token).json()
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\requests\models.py", line 975, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
pyrogram.dispatcher - Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\requests\models.py", line 971, in json
    return complexjson.loads(self.text, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python311\Lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\cw.py", line 109, in account_login
    html1 = s.get("https://elearn.crwilladmin.com/api/v1/comp/my-batch?&token=" + token).json()
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\requests\models.py", line 975, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
root - Stop signal received (SIGINT). Exiting...
__main__ - <---Bot Stopped-->
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 07:22:50.923229 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - cannot access local variable 'token' where it is not associated with a value
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\cw.py", line 106, in account_login
    await m.reply_text(f"```{token}```")
                       ^^^^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'token' where it is not associated with a value
root - Stop signal received (SIGINT). Exiting...
__main__ - <---Bot Stopped-->
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 07:24:29.002774 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - cannot access local variable 'token' where it is not associated with a value
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\cw.py", line 106, in account_login
    await m.reply_text(f"```{token}```")
                       ^^^^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'token' where it is not associated with a value
root - Stop signal received (SIGINT). Exiting...
__main__ - <---Bot Stopped-->
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 07:28:19.104692 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - cannot access local variable 'token' where it is not associated with a value
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\cw.py", line 107, in account_login
    await m.reply_text(f"```{token}```")
                       ^^^^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'token' where it is not associated with a value
root - Stop signal received (SIGINT). Exiting...
root - Stop signal received (SIGINT). Exiting...
__main__ - <---Bot Stopped-->
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-13 07:30:42.318332 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - cannot access local variable 'token' where it is not associated with a value
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\cw.py", line 107, in account_login
    await m.reply_text(f"```{token}```")
                       ^^^^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'token' where it is not associated with a value
pyrogram.connection.connection - Connecting...
pyrogram.connection.connection - Connected! Production DC5 - IPv4 - TCPAbridgedO
pyrogram.session.session - NetworkTask started
pyrogram.session.internals.msg_id - Time synced: 2023-04-14 08:48:24.139112 UTC
pyrogram.session.session - Session initialized: Layer 140
pyrogram.session.session - Device: CPython 3.11.3 - Pyrogram 1.4.16
pyrogram.session.session - System: Windows 10 (EN)
pyrogram.session.session - Session started
pyrogram.session.session - PingTask started
pyrogram.client - [StarkBot] [LOAD] MessageHandler("Start_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("restart_handler") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("log_msg") in group 0 from "plugins.__init__"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cp"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cpd"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.cw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.e1"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.exampur"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("forward") in group 0 from "plugins.forward"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.khan"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.mgconcept"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.ocean"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.pw"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.rjvikramjeet"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account") in group 0 from "plugins.sam"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.txt"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.vidya"
pyrogram.client - [StarkBot] [LOAD] MessageHandler("account_login") in group 0 from "plugins.winners"
pyrogram.client - [StarkBot] Successfully loaded 18 plugins from "plugins"
pyrogram.dispatcher - Started 50 HandlerTasks
__main__ - <--- @All_int_onebot Started (c) STARKBOT --->
pyrogram.dispatcher - cannot access local variable 'token' where it is not associated with a value
Traceback (most recent call last):
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyrogram\dispatcher.py", line 222, in handler_worker
    await handler.callback(self.client, *args)
  File "C:\Users\Rudraa\AppData\Roaming\Python\Python311\site-packages\pyromod\listen\listen.py", line 93, in resolve_listener
    await self.user_callback(client, message, *args)
  File "D:\Bot\all-in-bot\plugins\khan.py", line 67, in account_login
    'authorization': f"Bearer {token}",
                     ^^^^^^^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'token' where it is not associated with a value
